db_master_deploy/includes.sh

```diff
@@ -80,7 +95,14 @@ err() {
 #   integer
 #######################################
 Ceiling () {
-  python -c "from math import ceil; print int(ceil(float($1)/float($2)))"
+    DIVIDEND="${1}"
```

Comment:
Though the python version is simpler, getting rid of python entirely (since this was the only place it was used I think) will allow to create a much smaller docker image for shellspec. I'll adapt the docker image accordingly